cmd/livepeer/livepeer.go

```diff
@@ -227,3 +228,16 @@ func updateNilsForUnsetFlags(cfg starter.LivepeerConfig) starter.LivepeerConfig
 
 	return res
 }
+
+// Defaults session limit to "0" (auto) for orchestrators when -maxSessions flag not set
```

Comment:
How about being more explicit and having a value `--maxSessions auto`? Otherwise, I'm afraid of not keeping the backwards-compatibility for all cases.

Reply:
I have reverted back to -maxSessions auto instead of using `0`